src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx

```diff
@@ -51,6 +51,7 @@ const FixedContainer = styled.div<{ redesignFlag: boolean }>`
 `
 
 const Container = styled.div<{ hideInput: boolean; disabled: boolean; redesignFlag: boolean }>`
+  min-height: ${({ redesignFlag }) => redesignFlag && '69px'};
```

Comment:
do we want to make this a round number? either 68 or 72?

Reply:
69 is the needed number to make no shifts (due to the size of the other section)

Reply:
also, lolz :kayleedead:

Reply:
lmfaooo